### PR TITLE
Remaps Wasteplanet_tradepost + Adds The Concrete Caves

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request

remaps the shitty fucking wasteplanet ruin

adds concrete caves to wasteplanets.

https://github.com/shiptest-ss13/Shiptest/assets/94164348/815e4b59-d562-48f3-aa50-ca45f7e25315

## Why It's Good For The Game
i am the content slave and this shit is fucking awesome.

## Changelog

:cl:
add: wasteplanets now spawn concrete caves
add: Patrician-Class Trader Shuttle
add: wasteplanet_tradepost has been remade
add: acid proof marker beacons. it's self explanatory.

/:cl:

### CODE REVIEW

**Comment by meemofcourse** [comment]
fuck you baltimore!

**Comment by Latentish** [comment]
Poggers?
image

**Comment by ZephyrTFA** [comment]
Really do not appreciate getting an email titled "Fuck You" from github.
image

**Comment by goober3** [comment]
image

**Review by spockye — CHANGES_REQUESTED** [review]
image

atmos fuckies


image

cable fuckies

image

both doors and buttons use the same ID. 

image

no button for the engine shutters

image

powered holofans might be good to add here. 

could use some more decalling inside the building, could also use some kind of canteen/recreational room? Since its seems like a long term deployment

mabye add some extra threats in the main room, seeing that the loot is pretty good?
big hivebot.....


this is a massive improvement (and i like the setting)

**Comment by Erikafox** [comment]
>could use some more decalling inside the building, could also use some kind of canteen/recreational room? Since its seems like a long term deployment

It's intentionally pretty spartan, but I did add some mild "people ate here" greebling to a few spots. Review otherwise addressed

File: _maps/map_catalogue.txt [hunk 1]
```diff
@@ -595,8 +595,8 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Tags = "No Combat", "Medium Loot", "hospitable"
 
 		File name ="_maps\RandomRuins\wasteruins\wasteplanet_tradepost.dmm"
-		Size = (x = 15)(y = 20)(z = 1)
-		Tags "No combat", "Medium loot", "hospitable"
+		Size = (x = 57(y = 51(z = 1)
```
Comment (Latentish) [inline _maps/map_catalogue.txt]: It should be like (x = 57)(y = 51)(z = 1)

**Review by Latentish — CHANGES_REQUESTED** [review]

**Pull request closed without merging**